pkg/otlp/model/translator/metrics_translator.go

```diff
@@ -40,7 +40,7 @@ type Translator struct {
 func New(logger *zap.Logger, options ...Option) (*Translator, error) {
 	cfg := translatorConfig{
 		HistMode:                 HistogramModeNoBuckets,
-		SendCountSum:             true,
+		SendCountSum:             false,
```

Comment:
Should the default `HistMode` be `HistogramModeDistributions`? Otherwise the default config is illegal (I believe we throw an error if no buckets + no count/sum metrics)

Reply:
Updated to `HistogramModeDistributions`. 

> I thought we threw an error if no buckets + no count/sum metrics because that'd send nothing

We check that on the OpenTelemetry Collector configuration validation but not on the translator code. I can add that check, it makes sense to check here too.

Reply:
Added the check here too, thanks for catching that!